_pages/en/capabilities.md

```diff
@@ -471,31 +464,25 @@ InfoSec supplies development teams and operational teams with an ample number of
 
 #### 3.2.2. Indicators  
 
-- How often your team monitors the stability and security of your IT system?  
-   a. Very Frequently  
-   b. Frequently  
-   c. Occasionally  
-   d. Rarely  
-   e. Very Rarely  
-   f. Never  
+Ideal: Very Frequently  
 
-- How do you notify your team about failure alerts?  
-   a. Via email  
-   b. Via PR in Github  
-   c. Via Slack  
-   d. Via Trello board  
-   e. Via phone call  
-   f. None of the above/not applicable  
+- How often your team monitors the stability and security of your IT system?  
+  a. Very Frequently (Ideal)  
+  b. Frequently  
+  c. Occasionally  
+  d. Rarely  
+  e. Very Rarely  
+  f. Never  
 
 - How often do you  face the situation when your team finds out from customers that your application or service is down?  
   a. Always  
   b. Very Often  
   c. Sometimes  
   d. Rarely  
-  e. Never  
+  e. Never (Ideal)  
 
 - How long does it take to fix the problem?  
```

Comment:
- How long does it take to fix the problem?  (What problem?)
===>
- When a critical problem with the application is identified how often does it take, on average, for the fix to be on production?
Ideal: A day or less
More than a day, less than a week
Between one week and one month
More than a month